### ISSUE TITLE
Missing info module "Cache Management"

### ISSUE THREAD

**Issue by responseinformationsdesign** [AOEpeople/TYPO3-Cache-Management#7]
I can´t find the module "Cache Management" under Web > Info as mentioned in [https://docs.typo3.org/typo3cms/extensions/cachemgm/stable/Manual/Index.html#web-info-cache-management](url)

**Comment by robinson2** [comment]
Same for me

**Comment by pixelbrackets** [comment]
Looking at the Git history revealed that this functionality was removed in version 4.x - see commit https://github.com/AOEpeople/TYPO3-Cache-Management/commit/8e5de16f808aeabe7ae891a5dbabe8b5e8cc4a11

**Comment by pixelbrackets** [comment]
> Looking at the Git history revealed that this functionality was removed in version 4.x - see commit [8e5de16](https://github.com/AOEpeople/TYPO3-Cache-Management/commit/8e5de16f808aeabe7ae891a5dbabe8b5e8cc4a11)

Maybe @tomasnorre could give a hint why this was removed. I suppose the code base was to old to migrate to "newer" TYPO3 versions.

**Comment by tomasnorre** [comment]
It's some time ago, but if I remember correct, we/I removed it from the Info module and moved it to the "Systems" Section in the Left Menu in the backend. 

It was done as part of the refactoring and make it compatible with newer versions of TYPO3. 

The functionality should still be the same, but be located differently.

I have just tested it out in a new TYPO3 installation (v11), and not present either. 
Looks like a bug then. 

As I'm not using the Cachemanager personaly and not part of AOE any more. I will most likely not be the one providing the Fix. 

Perhaps @TomSchenk or @felixsemmler could help you out?
